src/components/settings/settings-form/settings-form.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Pane,
+  PaneCloseLink,
+  PaneFooter,
+  Button,
+} from '@folio/stripes/components';
+
+export default class SettingsForm extends Component {
+  static propTypes = {
+    children: PropTypes.node,
+    formState: PropTypes.object.isRequired,
+    onCancel: PropTypes.func.isRequired,
+    onSave: PropTypes.func.isRequired,
+    title: PropTypes.node,
+  };
+
+  renderFooter() {
+    const {
+      formState: {
+        form: { reset },
+        invalid,
+        pristine,
+        updateIsPending,
+      },
+    } = this.props;
+
+    const cancelButton = (
+      <Button
+        data-test-eholdings-settings-form-cancel-button
+        buttonStyle="default mega"
+        disabled={updateIsPending || pristine}
+        onClick={reset}
+        marginBottom0
+      >
+        <FormattedMessage id="stripes-components.cancel" />
+      </Button>
+    );
+
+    const saveButton = (
+      <Button
+        buttonStyle="primary mega"
+        data-test-eholdings-settings-form-save-button
+        disabled={updateIsPending || invalid || pristine}
+        marginBottom0
+        type="submit"
+      >
+        <FormattedMessage id="stripes-components.saveAndClose" />
+      </Button>
+    );
+
+    return (
+      <PaneFooter
+        renderStart={cancelButton}
+        renderEnd={saveButton}
+      />
+    );
+  }
+
+  render() {
+    const {
+      children,
+      title,
+      formState,
+      ...formProps
+    } = this.props;
+
+    return (
+      <form onSubmit={formState.handleSubmit} {...formProps}>
```

Comment:
Isn't  `handleSubmit`  destructured because destructuring would lead to losing `this`?